Dockerfile

```
@@ -47,7 +47,7 @@ ARG TARGETOS
 ARG TARGETARCH
 ARG TARGETVARIANT
 
-ENV GOMPLATE_VERSION=v3.11.8
+ENV GOMPLATE_VERSION=v4.0.0
```

```suggestion
ENV GOMPLATE_VERSION=v4.0.1
```
There is a new release already.

https://github.com/hairyhenderson/gomplate/releases/tag/v4.0.1

---

Since it was a breaking change release. I thought to better go with 4.0.0, in case community has more experience with. Anyway, I bumped it now to v4.0.1